beater/interceptors/logging.go

```diff
@@ -62,7 +62,7 @@ func Logging(logger *logp.Logger) grpc.UnaryServerInterceptor {
 			logger.With("error.message", res.Message()).Error(logp.Error(err))
 			return nil, err
 		}
-		logger.Info("request accepted")
+		logger.Debugf("request accepted")
```

Comment:
Controversial and might warrant its own PR but I found this to be very noisy just running with `-e`